lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -666,23 +690,23 @@ def write(chunk)
             end
 
           rescue Google::Apis::ServerError => error
-            # Server error, so retry via re-raising the error.
+            # 5xx Server errors, so retry via re-raising the error.
```

Comment:
For consistency: `5xx server errors. Retry via re-raising the error.`.

Reply:
Done.